ext/MitsubaLoader/CMitsubaLoader.h

```diff
@@ -40,17 +40,17 @@ namespace bsdf
 		uint32_t constant_f32;
 	};
 
-	static STextureData getTextureData(const ICPUImage* _img, ICPUTexturePacker* _packer)
+	static STextureData getTextureData(const asset::ICPUImage* _img, asset::ICPUTexturePacker* _packer)
 	{
 		STextureData texData;
 		core::vector2df_SIMD szUnorm16(_img->getCreationParameters().extent.width, _img->getCreationParameters().extent.height);
-		szUnorm16 /= core::vector2df_SIMD(_packer->maxAllocatableTextureSz());
+		szUnorm16 /= core::vector2df_SIMD(_packer->getPageTable()->getCreationParameters().extent.width);
```

Comment:
virtual texture size, not the page table size so `extent.width*pageSize`

Reply:
right, thanks